en/sensor/pmw3901.md

```diff
@@ -0,0 +1,148 @@
+The goal of the microcontroller is to then expose 2 UAVCAN messages containing all the necessary data.
+
+This board can be connected to the CAN port on any Pixhawk board.
+All tests were performed connecting it to the CAN1 port on a Pixhawk 4.
+Differently from the other sensors, this one needs UAVCAN running on PX4.
+To enable UAVCAN you will have to set the UAVCAN_ENABLE parameter to 2 (if you have have UAVCAN sensors but no motor controllers) or to 3 (if you have also your motor controllers on CAN).
+
+As for the other optical flow boards, it is recommended to use an external distance sensor instead of the onboard one due to the limited range of the VL53L1X.
+
+![Hex PH4 Pinout](../../assets/hardware/sensors/pmw3901/ph4-hex-optical-flow.png)
+TODO
+
+Regarding rotations, also this board will have a default orientation and if necessary this could be modified through a parameter.
+
+### Thone
+WORK IN PROGRESS
+
+### AlienTek
+
+WORK IN PROGRESS
+<!-- Minimal English specs here: https://www.aliexpress.com/item/32923971729.html -->
+
+![Alientek Pinout](../../assets/hardware/sensors/pmw3901/alientek.png)
+
+![Alientek Pixhawk4 Connections](../../assets/hardware/sensors/pmw3901/ph4-alientek-flow-pinout.png)
+
+
+## Mounting/Orientation {#orientation}
+
+The flow modules are typically mounted near the centre of the vehicle.
+If mounted off-centre you will need to set offsets: [Optical Flow > EKF2](../sensor/optical_flow.md#ekf2).
+
+> **Note** An external distance sensor may be mounted anywhere but should point down and be configured as usual.
+
+The flow modules may be mounted with any yaw orientation relative to vehicle body frame, but you must set the value used in [SENS_FLOW_ROT](../advanced_config/parameter_reference.md#SENS_FLOW_ROT).
+
+The "zero" rotation is when the board and [vehicle](../getting_started/px4_basic_concepts.md#heading-and-directions) X-axes are aligned (i.e. the "front" of the vehicle and the board are in the same direction), with rotations increasing in a clockwise direction.
+
+PMW3901 modules have a small notch indicating the direction of the **back** of the board. 
+The diagram shows the orientation of the Bitcraze board to the vehicle that corresponds to `SENS_FLOW_ROT=0` (note the notch at the back).
+
+![PMW3901 PH4 Rotation](../../assets/hardware/sensors/pmw3901/ph4-pmw3901-rotation.png)
+
+You can use the notch to find the orientation of the other boards in the same way:
+
+Tindie
+
+![PMW3901 Tindie Notch](../../assets/hardware/sensors/pmw3901/tindie_notch.jpg)
+
+Hex Hereflow
+
+![PMW3901 Hereflow Notch](../../assets/hardware/sensors/pmw3901/hex_hereflow_notch.jpg)
+
+Thone
+
+![PMW3901 Thoneflow Notch](../../assets/hardware/sensors/pmw3901/thoneflow_3901u_notch.jpg)
+
+Alientek (also has an arrow indicating the front!)
+
+![PMW3901 Alientek Notch](../../assets/hardware/sensors/pmw3901/alientek_pmw3901_notch.jpg)
+
+
+## Configuration {#configuration}
```

Comment:
I added this section @DanielePettenuzzo to test that my understanding is correct regarding config. Most important bit here is that the distance sensor requires no additional setup to work with flow.